torch/utils/data/datapipes/iter/filelister.py

```diff
@@ -1,12 +1,16 @@
 from typing import Iterator, List, Sequence, Union
 
+
+from torch.utils.data.datapipes._decorator import functional_datapipe
+
 from torch.utils.data.datapipes.datapipe import IterDataPipe
 from torch.utils.data.datapipes.iter import IterableWrapper
 from torch.utils.data.datapipes.utils.common import get_file_pathnames_from_root
 
 __all__ = ["FileListerIterDataPipe", ]
 
 
+@functional_datapipe("list_files")
```

Comment:
In a vacuum I would prefer `list_files` but for consistency, I agree with you.

@xiurobert Sorry, can you make this change?

Reply:
On one hand, it provides API consistency. On the other hand, it feels grammatically wrong.

Reply:
> In a vacuum I would prefer `list_files` but for consistency, I agree with you.

I have the same feeling...
How about we change all `DataPipe` in TorchData to `list_files`? I think we are fine to change them because these APIs are introduced after our last release.

Reply:
> I have the same feeling... How about we change all `DataPipe` in TorchData to `list_files`? I think we are fine to change them because these APIs are introduced after our last release.

Let's do that instead, especially since we have `open_files` for FileOpener.

Reply:
Made the changes in the lastest commit

Reply:
Let me open a PR in `torchdata`. We will have to check for domain libraries usages too.

Reply:
@NivekT Sounds great.